apps/dashboard/src/components/workflow-editor/steps/email/email-editor.tsx

```diff
@@ -16,20 +16,18 @@ export const EmailEditor = (props: EmailEditorProps) => {
 
   return (
     <div className="flex h-full flex-col">
-      <EmailTabsSection>
+      <div className="border-b px-4 pb-0 pt-4">
         <EmailPreviewHeader>
           {disableOutputSanitization &&
             getComponentByType({
               component: disableOutputSanitization.component,
             })}
         </EmailPreviewHeader>
-      </EmailTabsSection>
-      <EmailTabsSection className="min-h-[68px]">
         {getComponentByType({ component: subject.component })}
-      </EmailTabsSection>
-      <EmailTabsSection className="flex-1 bg-neutral-50 pl-16 pr-16 pt-5">
+      </div>
+      <div className="relative flex-1 overflow-y-auto bg-neutral-50 px-16 pt-8">
```

Comment:
Make only the Maily part scrollable, and keep the subject and the top section always visible. This UX feels better for the email editor when the content becomes lengthy.